gradlew

```diff
@@ -44,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
why do we need these?
64m is  enough?

Reply:
Just following default wrapper build outcome. If it doesn't suit to certain environment - can always amend. For future versions gradle might decide something else, follow other trends and so on and so forth